pkg/perfscaleanalysis/perfscalehelpers.go

```diff
@@ -0,0 +1,281 @@
+package perfscaleanalysis
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
+
+	es7 "github.com/elastic/go-elasticsearch/v7"
+	"k8s.io/klog"
+)
+
+const (
+	esURL       = "http://perf-results-elastic.apps.keith-cluster.perfscale.devcluster.openshift.com"
+	esIndex     = "perf_scale_ci"
+	esQuerySize = 2000
+)
+
+type ScaleJob struct {
+	UpstreamJob   string `json:"upstreamJob"`
+	ReleaseStream string `json:"releaseStream"`
+	NetworkType   string `json:"networkType"`
+	Platform      string `json:"platform"`
+
+	// Buckets contain the result totals for the configured periods of time.
+	Buckets []ScaleJobBucket `json:"buckets"`
+}
```

Comment:
@stbenjam not sure if we can shoehorn into JobRun, interested in your thoughts. Lack of a numeric ID is one problem, otherwise I think it's probably somewhat possible, but of course we don't have anywhere to store the bucket data.

Reply:
I think this is the right approach, instead of trying to force it into another type.

Will the API be for the scale job bucket instead of individual jobs? It's not entirely clear to me how we'll present the data to the user

Reply:
I was thinking it would be something like /scalejobs/ and the response contains the buckets embedded within. WDYT?